pkg/network/master/egressip.go

```diff
@@ -148,16 +152,17 @@ func (eim *egressIPManager) poll(stop chan struct{}) {
 		}
 
 		start := time.Now()
-		retry := eim.check(retry)
+		retry, _ := eim.check(retry)
```

Comment:
If you're going to return an error, you should handle it. Don't just add it only to drop it on the floor.